crates/astria-bridge-withdrawer/src/bridge_withdrawer/startup.rs

```diff
@@ -0,0 +1,613 @@
+use std::{
+    sync::Arc,
+    time::Duration,
+};
+
+use astria_core::{
+    bridge::Ics20WithdrawalFromRollupMemo,
+    primitive::v1::asset,
+    protocol::{
+        asset::v1alpha1::AllowedFeeAssetIdsResponse,
+        bridge::v1alpha1::BridgeAccountLastTxHashResponse,
+        transaction::v1alpha1::Action,
+    },
+};
+use astria_eyre::eyre::{
+    self,
+    ensure,
+    eyre,
+    Context as _,
+    ContextCompat,
```

Comment:
`eyre` actively discourages the use of this and only provides this as a crutch to move from anyhow, see here: https://docs.rs/eyre/latest/eyre/trait.ContextCompat.html#why-doesnt-eyre-impl-wraperr-for-option

Please change this.